RELEASE.md

```diff
@@ -21,6 +21,70 @@ The binaries releases for HEAD are available here:
 - [kubebuilder-release-master-head-darwin-amd64.tar.gz](https://storage.googleapis.com/kubebuilder-release/kubebuilder-release-master-head-darwin-amd64.tar.gz)
 - [kubebuilder-release-master-head-linux-amd64.tar.gz](https://storage.googleapis.com/kubebuilder-release/kubebuilder-release-master-head-linux-amd64.tar.gz)
 
+## How the releases are configured
+
+The releases occur in an account in the Google Cloud (See [here](https://console.cloud.google.com/cloud-build/builds?project=kubebuilder)) using Cloud Build.
+
+### To build the Kubebuilder CLI binaries:
+
+A trigger `build-kb-release` is configured to call [build/cloudbuild.yaml](build/cloudbuild.yaml).
+This trigger will be executed when any new tag be published. 
+The tags must be built from the release branch (Currently, `release-3`).
+
+Also, we have a trigger to generate snapshots builds from the master branch.
+This trigger will call [build/cloudbuild_snapshot.yaml](build/cloudbuild_snapshot.yaml)
+when any change needs to be performed on master.
+
+### To build the Kubebuilder-tools: (Artifacts required to use ENV TEST)
+
+Kubebuilder projects requires artifacts which are used to do test with ENV TEST (when we call `make test` target)
+These artifacts can be checked in the service page: https://storage.googleapis.com/kubebuilder-tools
+
+The build is made from the branch [tools-releases](https://github.com/kubernetes-sigs/kubebuilder/tree/tools-releases) and the trigger will call the `build/cloudbuild_tools.yaml` passing 
+as argument the architecture and the SO that should be used, e.g:
+
+<img width="553" alt="Screenshot 2022-04-30 at 10 15 41" src="https://user-images.githubusercontent.com/7708031/166099666-ae9cd2df-73fe-47f6-a987-464f63df9a19.png">
```

Comment:
Should this screenshot-png be part of the PR? I am not sure how images are handled in the docs. But could not find any [existing references](https://github.com/kubernetes-sigs/kubebuilder/search?q=user-images.githubusercontent.com).

Reply:
If we click on the view, we can check that. This doc is not used in the books. 
So that is fine. I mean, it works fine see:

<img width="481" alt="Screenshot 2022-05-09 at 13 26 56" src="https://user-images.githubusercontent.com/7708031/167409646-e624b768-f1c1-4377-8f70-5d242a98f14c.png">

<img width="1039" alt="Screenshot 2022-05-09 at 13 27 03" src="https://user-images.githubusercontent.com/7708031/167409658-a1516b6a-56d1-4ca5-8440-d0ddce32464a.png">